scripts/gen-universe.py

```diff
@@ -409,17 +411,12 @@ def write_package_in_zip(zip_file, path, package):
     :rtype: None
     """
 
-    package = copy.deepcopy(package)
+    package = downgrade_package_to_v2(package)
+
     package.pop('releaseVersion')
-    package.pop('minDcosReleaseVersion', None)
-    package['packagingVersion'] = "2.0"
 
     resource = package.pop('resource', None)
```

Comment:
Won't `downgrade_package_to_v2()` have already removed this field, because it calls `v3_to_v2_package()`?